src/components/QandA/QandA.jsx

```diff
@@ -1,64 +1,263 @@
+    setCount(count => ({ ...count, answers: count.answers + 1 }));
+    // console.log('results = ', results);
+    // console.log('count = ', count);
+    // console.log('showData = ', showData);
   }
 
+  const moreAnsweredQuestions = () => {
+    // console.log('results = ', results)
+    // console.log('showData = ', showData)
+    if (results.length === questionCount) {
+      return console.log('No more questions!')
+    }
+    setQuestionCount(questionCount + 1);
+  }
 
-  return (
-    <div className="QandA">
-      <div className="Search">
-      <h2> QUESTIONS & ANSWERS (count = {count})</h2>
-      <input
-        className="search-bar"
-        type="Text"
-        onChange={data}
-        placeholder="Have a question? Search for answers ..."
-      />
-      </div>
-      <QuestionList count={count} questions={questions}/>
-      <br></br>
-      <button id="load" onClick={loadMoreAnswers}>
-        <b>LOAD MORE ANSWERS</b></button><br></br>
-      <button id="moreQA">
-        <b>MORE ANSWERED QUESTIONS</b></button>
-      <button id="addQ">
-        <b>ADD A QUESTION +</b></button>
-    </div>
-  )
+  // for debugging purposes >.<
+  if (0 === 0) {
+    return (
+    <>
+      <h2> {'QUESTIONS & ANSWERS'}</h2>
+      <Sort>
+        <Search>
+          <Input
+            src={MagGlass}
+            type="Text"
+            onChange={(e) => {
+              setSearch(e.target.value);
+            }}
+            placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..."
+            value={search}>
+          </Input>
+          <Icon src={MagGlass}/>
+        </Search>
+        <Questions>
+        {showData.filter((question) => {
+            if (search.length < 3) {
+              return question;
+            } else if (question.question.toLowerCase().includes(search.toLowerCase())) {
+              return question;
+            }
+          }).slice(0, questionCount).map((question, index) => {
+            return (
+              <div key={index}>
+                <h3>Q: {question.question}</h3>
+                  {question.answers.map((answer, index) => {
+                    return (
+                      <div key={index}>
+                        <h3 style={{display: 'inline'}}>A: </h3>
+                        <p style={{display: 'inline'}}>{answer.body}</p>
+                        <pre>   by {answer.answerer_name}, {answer.date.slice(0, 10)}   |   Helpful? <u>Yes</u>({answer.helpfulness})   |   <u>Report</u></pre>
+                      </div>
+                    )
+                  })}
+              </div>
+            )
+          })}
+        </Questions><br></br>
+        <button id="load" onClick={loadMoreAnswers}>
+          <b>LOAD MORE ANSWERS</b></button><br></br><br></br>
+        <Button onClick={moreAnsweredQuestions}>
+          <b>MORE ANSWERED QUESTIONS</b>
+        </Button>
+        <Button>
+          <b>ADD A QUESTION +</b>
+        </Button>
+      </Sort>
+    </>
+    )
+  }
 };
 
```

Comment:
are you having to do this because it isn't rendering? for this part maybe you can wrap the divs with brackets {results && <><h2....etc.}. this means that if results is a truthy value it will render anything after &&. I've had to do this for some of my stuff too.

Reply:
Thanks, I will try that out!